src/plot_erpimage.jl

```diff
@@ -52,6 +57,7 @@ function plot_erpimage!(
             sortindex = 1:size(plot, 2)
         else
             sortindex = sortperm(sortvalues)
+            ax.ylabel = "Trials sorted" #by $(mymacro(sortvalues))
```

Comment:
this should be specified via config, else it is not customizable

Reply:
the idea is that in config there is already "Trials"
While here is changed to "Trials sorted"
I also wanted to include by which variable it is sorted

Reply:
mh, but your solution makes it impossible to overwrite that label via a custom `axis=(;ylabel="My Label is better")`

You could check whether in the kwargs axis.title is empty or not and then use your default maybe